### ISSUE TITLE
Recognize curly quote as apostrophe

### ISSUE THREAD

**Issue by rrthomas** [entorb/typonuketool#2]
The Unicode ’ should be recognized (at least optionally) as an apostrophe, that can be part of a word.

**Comment by entorb** [comment]
Yes you are right, but since people tend to use different (wrong) characters for this sign (`’‘') and because I do not plan to put more time into this old tool, I would rather not modify it
